library/src/layout/enum.rs

```diff
@@ -132,6 +132,27 @@ pub struct EnumElem {
     /// If set to `{auto}`, uses the spacing [below blocks]($func/block.below).
     pub spacing: Smart<Spacing>,
 
+    /// The horizontal alignment that enum numbers should have, or `{auto}` to
+    /// inherit the text alignment from the context.
+    ///
+    /// By default, this is set to `{end}`, which aligns enum numbers
+    /// towards end of the current text direction (in left-to-right script,
+    /// for example, this is the same as `{right}`). The choice of `{end}`
+    /// for horizontal alignment of enum numbers is usually preferred over
+    /// `{start}`, as numbers then grow away from the text instead of towards
+    /// it, avoiding certain visual issues. This option lets you override this
+    /// behavior, however.
+    ///
+    /// ````example
+    /// #set enum(number-align: start)
+    ///
+    /// 1.  Numbers here are start-aligned,
+    /// 9.  overriding the default of
+    /// 10. end-aligned.
```

Comment:
This example might make people think that the source formatting is relevant somehow (because the source numbers are also start-aligned). Not sure whether I like that better, but we can also make a long list only with `+` and reach ten. It's also a bit strange how the list items form a sentence.

Reply:
Good observations, thanks. How about now?

Reply:
Nice!